Tests/Scenarios/Server_Scenario_03_Remove.ps1

```diff
@@ -24,7 +58,7 @@ Configuration Server_Scenario_03_Remove
             Name = "OctopusServer"
 
             # The url that Octopus will listen on
-            WebListenPrefix = "http://localhost:81"
+            WebListenPrefix = "http://localhost:81,https://localhost"
```

Comment:
Do we even need this line? If we're removing the instance, then telling it what urls to listen on is a bit weird.